main.js

```diff
@@ -3,4 +3,6 @@ require('babel-register')({
     presets: ['env']
 });
 
-module.exports = require('./test.js')
\ No newline at end of file
+module.exports = require('./test.js')
+module.exports = require('./patherTests.js')
+module.exports = require('./mobSpawnerTests.js')
```

Comment:
Might want to look into using a .babelrc config file.
You can use babel-node for development and then using a babel script for building. 
Take a look at [engine repo](https://github.com/Project-Rekt/engine) at the .babelrc and package.json scripts.